core/cmd/local_client.go

```diff
@@ -139,7 +139,12 @@ func (cli *Client) RunNode(c *clipkg.Context) error {
 	if e := app.Start(); e != nil {
 		return cli.errorOut(fmt.Errorf("error starting app: %+v", e))
 	}
-	defer func() { lggr.ErrorIf(app.Stop(), "Error stopping app") }()
+	defer func() {
+		lggr.ErrorIf(app.Stop(), "Error stopping app")
+		if err = lggr.Sync(); err != nil {
+			panic(err)
```

Comment:
`log.Println` here too?
```suggestion
			log.Println(err)
```